File: SofiForce.Models/Models/EntityModels/OperationRequestModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class OperationRequestModel
    {
        public int OperationId { get; set; }
        public string OperationCode { get; set; }
        public int? AgentId { get; set; }
        public int? OperationTypeId { get; set; }
        public int? RepresentativeId { get; set; }
        public int? GovernerateId { get; set; }

        
        public DateTime? OperationDate { get; set; }
        public DateTime? StartDate { get; set; }
        public int? TargetDays { get; set; }
        public int? ActualDays { get; set; }
        public int? TargetClients { get; set; }
        public int? ActualClients { get; set; }
        public decimal? DaysPerformance { get; set; }
        public decimal? ClientsPerformance { get; set; }
        public decimal? Accuracy { get; set; }
        public string MapPoints { get; set; }
        
        public bool? IsClosed { get; set; }
        public DateTime? CloseDate { get; set; }
        public int? CBy { get; set; }
        public int? EBy { get; set; }
        public DateTime? CDate { get; set; }
        public DateTime? EDate { get; set; }

        public string Notes { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/OperationRequestModel.ts
export class OperationRequestModel {
    operationId: number;
    operationCode: string;
    agentId: number | null;
    operationTypeId: number | null;
    governerateId: number | null;
    representativeId: number | null;
    representativecode:string |'';
    operationDate: Date | null;
    startDate: Date | null;
    targetDays: number | null;
    actualDays: number | null;
    targetClients: number | null;
    actualClients: number | null;
    daysPerformance: number | null;
    clientsPerformance: number | null;
    accuracy: number | null;
    mapPoints: string;
    isClosed: boolean | null;
    closeDate: Date | null;
    notes:string |null;
    cBy: number | null;
    eBy: number | null;
    cDate: Date | null;
    eDate: Date | null;
}